pkg/odo/cli/component/create.go

```diff
@@ -543,8 +543,15 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 			}
 		}
 
-		if componentNamespace == "default" {
-			log.Warning("odo may not work as expected in a default project, please run the odo component in a non-default project. To create a new project, use `odo project create`.")
+		// Check whether resource "Project" is supported
+		projectSupported, err := co.Client.IsProjectSupported()
+
+		if err != nil {
+			return errors.Wrap(err, "Resource project validation check failed.")
```

Comment:
please start the error message with a small letter

Reply:
Done.